### ISSUE TITLE
Hosted fields have `inputmode=numeric' attribute

### ISSUE THREAD

**Issue by aderaaij** [braintree/braintree-web#673]
### General information

- SDK version: 3.91.0.
- Environment: Prod
- Browser and OS: Any

### Issue description

Currently all of the hosted fields have the `inputmode="number"` attribute on them, resulting in phone users not being able to correctly fill in the cardholder name and postal code. This is how the cardholder name field is rendered in the iframe: 
```html
<input autocomplete="cc-name" type="text" autocorrect="off" autocapitalize="none" spellcheck="false" class="cardholderName" data-braintree-name="cardholderName" name="cardholder-name" id="cardholder-name" inputmode="numeric" placeholder="John Doe" maxlength="255" aria-describedby="field-description-cardholderName">
```

Is there any way we can get this fixed? I've tried up/downgrading the braintree-web version and using `hostedFieldsInstance.setAttribute` and `hostedFieldsInstance.removeAttribute` to no avail. 

edit: pinning an older version seems to work, still would be nice to have the most recent version working

**Comment by SancSalix** [comment]
If you pass in a type while creating hosted fields, it won't assign inputmode to that field, but I am going to file a ticket for that bug and get it worked on as well

**Comment by lol768** [comment]
This is frankly unacceptable that Braintree have known this has been fucked up for over 3 weeks and haven't sorted it out. Cardholder name is broken out of the box with the most recent version of the Dropin UI on NPM - customers simply can't pay by card using an Android or iOS phone because the validation rejects a numeric cardholder name.

Stunning levels of incompetence here.

**Comment by hollabaq86** [comment]
This was fixed in 3.92.1.

Also as a heads up, **we will not tolerate abusive comments in our Github issues and pull request conversations.** If you can't treat the maintainers of our repos with respect, we will block you from future interactions.

**Comment by lol768** [comment]
> Also as a heads up, we will not tolerate abusive comments in our Github issues and pull request conversations. If you can't treat the maintainers of our repos with respect, we will block you from future interactions.

This isn't a volunteer community project. You are a service provider being paid to provide a payment gateway service.

This entire interaction, the lack of urgency with which it has been handled, the fact that such a fundamental issue was introduced in the first place has only prioritised our move to Stripe.
